website/source/docs/agent/services.html.md

```diff
@@ -93,39 +94,92 @@ meta object in node definition.
 All those meta data can be retrieved individually per instance of the service
 and all the instances of a given service have their own copy of it.
 
-The `kind` field is used to optionally identify the service as an [unmanaged
-Connect proxy](/docs/connect/proxies.html#unmanaged-proxies) instance with the
-value `connect-proxy`. For typical non-proxy instances the `kind` field must be
-omitted. The `proxy` field is also required for unmanaged proxy registrations
-and is only valid if `kind` is `connect-proxy`. The only required `proxy` field
-is `destination_service_name`. From version 1.2.0 to 1.3.0 this was specified
-using `proxy_destination` which still works but is now deprecated. See the
-[unmanaged proxy
-configuration](/docs/connect/proxies.html#complete-configuration-example)
-documentation for full details.
+Services may also contain a `token` field to provide an ACL token. This token is
+used for any interaction with the catalog for the service, including
+[anti-entropy syncs](/docs/internals/anti-entropy.html) and deregistration.
+
+A service can have an associated health check. This is a powerful feature as
+it allows a web balancer to gracefully remove failing nodes, a database
```

Comment:
Heh no that's odd phrasing but good catch since htis is merge fail - this is copy-paste from the ### Checks section below which didn't change. I'll fix it up.